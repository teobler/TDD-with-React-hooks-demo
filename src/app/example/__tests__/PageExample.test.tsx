import {PageExample} from "../PageExample";
import {render} from "@testing-library/react";
import * as React from "react";

describe('PageExample', () => {
    it('should render correctly', () => {
        const {container} = render(<PageExample/>);
        expect(container).toMatchSnapshot();
    });
});