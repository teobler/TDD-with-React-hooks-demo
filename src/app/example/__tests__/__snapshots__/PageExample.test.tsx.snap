// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageExample should render correctly 1`] = `
<div>
  <div>
    <form>
      <input
        type="text"
      />
    </form>
    <button>
      test
    </button>
  </div>
</div>
`;
